ractor.c

```diff
@@ -2237,6 +2237,28 @@ struct cross_ractor_require {
     ID name;
 };
 
+static void
+cross_ractor_require_mark(void *ptr)
+{
+    struct cross_ractor_require *crr = (struct cross_ractor_require *)ptr;
+    rb_gc_mark(crr->port);
+    rb_gc_mark(crr->result);
+    rb_gc_mark(crr->exception);
+    rb_gc_mark(crr->feature);
+    rb_gc_mark(crr->module);
+}
```

Comment:
Should we use declarative marking for this instead? https://github.com/Shopify/ruby/blob/master/doc/extension.rdoc/#declaratively-markingcompacting-struct-references-